### ISSUE TITLE
Multi-adds

### ISSUE THREAD

**Issue by duweihua** [xiaom233/BSRN#7]
How to get the number of Multi-adds and parameters? I use torchstat can not get the value in paper

**Comment by wxslby** [comment]
I have encountered same question.Could you solve it ?

**Comment by duweihua** [comment]
Using thop  tool



---- Replied Message ----
| From | ***@***.***> |
| Date | 10/19/2022 14:57 |
| To | ***@***.***> |
| Cc | ***@***.******@***.***> |
| Subject | Re: [xiaom233/BSRN] Multi-adds (Issue #7) |

I have encountered same question.Could you solve it ?

**Comment by Yzs007** [comment]
> Using thop tool
> […](#)
> ---- Replied Message ---- | From | ***@***.***> | | Date | 10/19/2022 14:57 | | To | ***@***.***> | | Cc | ***@***.******@***.***> | | Subject | Re: [xiaom233/BSRN] Multi-adds (Issue #7) | I have encountered same question.Could you solve it ? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***>

Hello，can you show me how you did it?

**Comment by duweihua** [comment]
> > Using thop tool
> > […](#)
> > ---- Replied Message ---- | From | _**@**_._**> | | Date | 10/19/2022 14:57 | | To | _**@**_.**_> | | Cc | _**@**_._**_**@**_.**_> | | Subject | Re: [xiaom233/BSRN] Multi-adds (Issue #7) | I have encountered same question.Could you solve it ? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: _**@**_.***>
> 
> Hello，can you show me how you did it?

No, I  also can not get the same value on paper with thop. Multi-adds(19.2) is always smaller than the paper results(19.4).

**Comment by duweihua** [comment]
Multi-adds(19.2) is always smaller than the paper results(19.4) with thop or torchsummaryX. Why? @xiaom233 
![image](https://user-images.githubusercontent.com/43801202/222099830-3480cc3f-0570-4ab4-99de-f1613b7f741a.png)

**Comment by xiaom233** [comment]
> Multi-adds(19.2) is always smaller than the paper results(19.4) with thop or torchsummaryX. Why? @xiaom233 ![image](https://user-images.githubusercontent.com/43801202/222099830-3480cc3f-0570-4ab4-99de-f1613b7f741a.png)

Please refer to the latest update, run
python basicsr/summary.py
